.evergreen/run-tests.sh

```diff
@@ -13,9 +13,10 @@ export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
 
 case $1 in
   "node")
-    npm run test-node
+    npm run check:coverage
```

Comment:
Getting out in front of the changes in coming in the many future ticket, I wanna make sure we don't drop as tests might get some minimal refactoring along with src as needed.